### PR TITLE
Add a page for rubygems.org/api/v2

### PR DESCRIPTION
**Pull request by fotanus**
The first endpoint was include in rubygems/rubygems.org#980

### CODE REVIEW

**Comment by arthurnn** [comment]
@qrush ?

File: rubygems-org-api-v2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> NOTE: This API version is under construction! It might change without further notice.
+
+Gem Version Methods
+-------------------
+
+### GET - `/api/v2/rubygems/[GEM NAME]/versions/[VERSION NUMBER].(json|yaml)`
+
+Returns a dictionary with versions details for a specific gem version. Example:
+
+    $ curl https://rubygems.org/api/v1/versions/coulda.json
```
Comment (qrush) [inline rubygems-org-api-v2.md]: `v1` should be `v2`

File: specification-reference.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ next: /command-reference
 
 
 <p>The Specification class contains the
-information for a Gem.  Typically defined in a .gemspec file or a Rakefile,
-and looks like this:</p>
+information for a <a href="../Gem.html">Gem</a>.  Typically defined in a
```
Comment (qrush) [inline specification-reference.md]: This page doesn't exist - why were there changes to this file?

Comment (fotanus) [inline specification-reference.md]: I actually have no idea :-)
Totally not remember of doing this myself.

File: rubygems-org-api-v2.md (hunk 1 above)
Comment (fotanus) [inline rubygems-org-api-v2.md]: Right, it actually was the wrong URL. fixed.

**Comment by olivierlacan** [comment]
@qrush Seems like the PR was updated according to your comments.

**Comment by fotanus** [comment]
I can update this PR if something is missing

**Comment by sonalkr132** [comment]
@fotanus `previous:` of [run-your-own-gem-server.md](https://github.com/rubygems/guides/blob/gh-pages/run-your-own-gem-server.md) would change to `/rubygems-org-api-v2` as well?

**Comment by fotanus** [comment]
Right, I tested here and makes sense. Fixed this previous.

**Comment by duckinator** [comment]
Can this be merged?

**Comment by fotanus** [comment]
@segiddins is this docs up-to-date?

**Comment by segiddins** [comment]
@fotanus can we merge this?

**Comment by fotanus** [comment]
Yes, merge it! It is up to date!

**Pull request merged**